mysql-test/include/not_rocksdb_dd.inc

```diff
@@ -0,0 +1,3 @@
+if (`select count(*) = 1 from performance_schema.global_variables where variable_name = 'default_dd_storage_engine' and variable_value = 'RocksDB';`) {
```

Comment:
@luqun, I think it makes sense to split this one file to its own separate PR and merge ASAP. We can already start annotating tests with it.

(I'd also consider replacing negative with positive i.e. `requires_innodb_ddse` in the filename and same in the check itself)

Reply:
It looks like it also makes sense to have `have_innodb_ddse_unless_row_binlog.inc`, to address testcases that try to replicate administrative statements in row format, thus require MyRocks to replicate in row. I'll work on this some more.

Reply:
> @luqun, I think it makes sense to split this one file to its own separate PR and merge ASAP. We can already start annotating tests with it.
> 
> (I'd also consider replacing negative with positive i.e. `have_innodb_ddse` in the filename and same in the check itself)

how about https://github.com/facebook/mysql-5.6/pull/1344? also Let me update this PR